### DISCUSSION TITLE
til-site/posts/selkouutiset-archive-now-supports-77-languages/

### DISCUSSION THREAD

**Discussion by giscus** [hiAndrewQuinn/til#10] [Announcements]
# til-site/posts/selkouutiset-archive-now-supports-77-languages/

Example from yesterday’s news. Count ’em yourself – there’s 76 of them there.
Mass i18n efforts like this are I think an underappreciated benefit of what static site generators like Hugo can give you. Actually, especialy Hugo – it’s multi-language support is very good, like darn near everything about the platform once you get past the initial learning curve.
Another underappreciated benefit: When building HTML pages is fast, you can afford to build a lot of them.

https://hiandrewquinn.github.io/til-site/posts/selkouutiset-archive-now-supports-77-languages/

**Comment by hiAndrewQuinn** [comment]
*Google Cloud Platform free trial, not Cloud Translation free trial.

(Which is quite generous! $300 USD of credit + another $100 for institutions for the first 90 days!)
